docs/index.md

```diff
@@ -0,0 +1,22 @@
+# Charmed Karapace Documentation
+
+The Charmed Karapace Operator delivers automated operations management from day 0 to day 2 on [Karapace](https://www.karapace.io/).
+
+Karapace provides an open source alternative to Confluent's [Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html#sr-overview). It is a solution to enforce data validation, consistency, serialization/deserialization... It is used as an additional service by producers/consumers to support data operations.
+
+As Karapace requires a running Kafka cluster, this operator makes use of the [Kafka Operator](https://github.com/canonical/kafka-operator) in order to work.
+
+## Project and community
+
+Charmed Karapace is an open-source project that welcomes community contributions, suggestions, fixes and constructive feedback.
+- [Read our Code of Conduct](https://ubuntu.com/community/code-of-conduct)
+- [Join the Discourse forum](/tag/karapace)
+- [Contribute](https://github.com/canonical/karapace-operator/blob/main/CONTRIBUTING.md) and report [issues](https://github.com/canonical/karapace-operator/issues/new)
+- Explore [Canonical Data Fabric solutions](https://canonical.com/data)
+- [Contact us]([/t/13107) for all further questions
+
+## License
+
+The Charmed Karapace Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/canonical/karapace-operator/blob/main/LICENSE) for more information.
+
+# Contents
```

Comment:
```suggestion
```

Reply:
Uhm, I believe this is placed by the automation as it does not seem to be there on Discourse. I believe it is because the don't have any item in the navigation table, as that's what should be placed below this headline. 

I believe it is a small improvement in the gatekeeper logic. could you submit an issue in the gatekeeper repo?

Reply:
> Uhm, I believe this is placed by the automation as it does not seem to be there on Discourse. 

But I [see it on Discourse](https://discourse.charmhub.io/t/charmed-karapace-documentation/14178) in the form of the Navigation section, which is empty. Can we delete it?

Also, the behavior is slightly different in other charms, for example:

There is non-empty Nabigation on the Discourse post - https://discourse.charmhub.io/t/charmed-kafka-documentation/10288
https://github.com/canonical/kafka-operator/blob/main/docs/index.md -- there is no Contents section in the `docs/index.md` file

Reply:
Added the Issue to the gatekeeper repo - https://github.com/canonical/discourse-gatekeeper/issues/251

But I still wonder whether we can delete the Empty Navigation section on Discourse.